.github/workflows/test.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Test
+
+on: push
+
+jobs:
+  manager:
+    name: Build and Test
+    
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # TODO run on Windows as well
+        os: [ubuntu-latest, macos-latest]
+      # Don't cancel jobs on other platforms if one fails
+      fail-fast: false
+      
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v1
+
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Show Environment Info
+        run: yarn -v
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Check yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+      
+      - name: Install dependencies
+        run: yarn --prefer-offline
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Manager
+        run: |
+          yarn do server_manager/electron_app/build
+          yarn do server_manager/web_app/build
```

Comment:
This is already called by yarn do server_manager/electron_app/build, I think.

Reply:
Done in dfb1a76